gpbft/participant.go

```diff
@@ -485,7 +485,16 @@ func (p *Participant) fetchCommittee(instance uint64) (*committee, error) {
 		if err := power.Validate(); err != nil {
 			return nil, fmt.Errorf("instance %d: %w: invalid power: %w", instance, ErrValidationNoCommittee, err)
 		}
-		comt = &committee{power: power, beacon: beacon}
+
+		// TODO: filter out participants with no effective power after rounding?
```

Comment:
This could save us a little perf, but it won't matter much and it'll make things more complex. IMO, don't do it.

Reply:
Nah.